lib/google_crawler_web/views/api/user_token_view.ex

```diff
@@ -0,0 +1,11 @@
+defmodule GoogleCrawlerWeb.Api.UserTokenView do
+  use JSONAPI.View, type: "user_token"
+
+  def fields do
+    [:id, :access_token]
```

Comment:
Just wondering what is the ID. Is it an ID of `UserToken`?

Reply:
Yea, but it is not needed in the attributes. I removed it